### DISCUSSION TITLE
Tabs overflow-x-auto

### DISCUSSION THREAD

**Discussion by mimoudix** [tailwindlabs/tailwindcss#12100] [Help]
![IMG_3698](https://github.com/tailwindlabs/tailwindcss/assets/13495877/9bfa6a78-549e-4675-a72e-f021c22e9522)

Everything works fine here, Im dealing with tabs panel and i need some help in UX. Is there anything i can do to let visitor know that this section is scrollable. Most of people wont notice this and I can't really use the dropdown used in tailwindUI.

**Comment by alex-krasikau** [comment]
The commonly used pattern here is to display a slight gradient or shadow on the sides. Here are some examples from Google:
- [Example 1](https://codepen.io/kidd1118/pen/qxQwvE)
- [Example 2](https://css-tricks.com/books/greatest-css-tricks/scroll-shadows/)

You can also take a look at the desktop version of YouTube. In addition to the shadow, it also displays arrows.

**Reply by mimoudix** [reply]
Ooh, nice one mate.  I will try it 👍

**Reply by mimoudix** [reply]
I did it, but seems something wrong about it (I didnt like it ) https://play.tailwindcss.com/OSsoJ8vOFI

**Reply by mimoudix** [reply]
@adamwathan please 🙏 What can we do it to make it enjoyable Master https://play.tailwindcss.com/OSsoJ8vOFI? ?

**Reply by alex-krasikau** [reply]
You can try [hiding the scollbar](https://www.w3schools.com/howto/howto_css_hide_scrollbars.asp) and trying different configuration of the shadows to look better.
